openvpn/protonvpn/update-port.sh

```diff
@@ -55,9 +55,12 @@ bind_trans() {
 		echo "setting transmission port to $new_port"
 		transmission-remote ${TRANSMISSION_RPC_PORT} ${myauth} -p "$new_port"
 
-		echo "Checking port..."
-		sleep 10
-		transmission-remote ${TRANSMISSION_RPC_PORT} ${myauth} -pt
+		echo "Waiting for port..."
+		until [[ $(transmission-remote ${TRANSMISSION_RPC_PORT} ${myauth} -pt | grep -ioE 'yes|no' | tr '[:upper:]' '[:lower:]') == "yes" ]]; do
```

Comment:
I suggests some kind of exit clause in case the port never is opened etc, like 5 tries and then error or such

Reply:
I added some retry functionality. I set retries to 50, I've seen it take a few minutes to finally open.